db/db_test.cc

```diff
@@ -3250,6 +3250,8 @@ class ModelDB : public DB {
 
   void NewManifestOnNextUpdate() override {}
 
+  uint64_t GetNextFileNumber() const override { return 0; };
```

Comment:
local test compilation fails without this. I believe we previously forgot to add this function in `ModelDB`.